charts/qfusion/v3.12.0/questions.yml

```diff
@@ -0,0 +1,28 @@
+rancher_max_version: 2.9.99
```

Comment:
This code is not needed if there are no version restrictions

Reply:
removed